docs/source/user-manual/copy-paste-blocks.md

```diff
@@ -0,0 +1,46 @@
+---
+myst:
+  html_meta:
+    "description": "Copy, cut, and paste blocks in Volto, the Plone 6 frontend."
+    "property=og:description": "Copy, cut, and paste blocks in Volto, the Plone 6 frontend."
+    "property=og:title": "How to copy, cut, and paste Volto blocks"
+    "keywords": "Volto, Plone, frontend, React, User manual, edit blocks, copy, cut, paste"
+---
+
+(copy-cut-paste-blocks-label)=
+
+# Copy, Cut, and Paste blocks in Volto
+
+Volto supports copying and cutting of blocks from one page, and pasting them to another page.
+This chapter describes how to do so.
+
+(copy-cut-blocks-label)=
+
+## Copy/Cut blocks
+
+While in edit mode you can hold {kbd}`ctrl` key to select multiple blocks you want to copy/cut.
+Once blocks are selected, you will have a copy <img alt="../_static/copy.svg" src="../_static/copy.svg" class="inline"> and a cut <img alt="../_static/cut.svg" src="../_static/cut.svg" class="inline"> option at the bottom of Toolbar.
```

Comment:
The toolbar goes horizontal and to the top of the window on small screens.

```suggestion
Once blocks are selected, you will have copy <img alt="../_static/copy.svg" src="../_static/copy.svg" class="inline"> and cut <img alt="../_static/cut.svg" src="../_static/cut.svg" class="inline"> options in the toolbar.
```

Reply:
Oh yeah I missed that , Thanks for pointing it out.